tests/shell/shell_recorder/listtest.esr

```diff
@@ -16,4 +16,5 @@ RET: 0
 
 < kdb ls $Mountpoint
 
+< kdb rm /sw/kdb/current/plugins
```

Comment:
This is dangerous, it will remove user data.

Ideally tests should only manipulate `/test` or at least restore other changes.